src/item_contents.cpp

```diff
@@ -102,11 +102,19 @@ pocket_favorite_callback::pocket_favorite_callback( std::vector<item *> to_organ
 void pocket_favorite_callback::add_pockets( item &i, uilist &pocket_selector,
         std::string depth )
 {
-    if( !i.get_all_contained_pockets().empty() ) {
-        pocket_selector.addentry( -1, false, '\0', string_format( "%s%s", depth, i.display_name() ) );
-        // pad list with empty entries for the items themselves
-        saved_pockets.emplace_back( nullptr, 0, nullptr );
+    if( i.get_all_contained_pockets().empty() ) {
+        // if it doesn't have pockets skip it
+        return;
     }
+
+    pocket_selector.addentry( -1, false, '\0', string_format( "%s%s", depth, i.display_name() ) );
+    // pad list with empty entries for the items themselves
+    saved_pockets.emplace_back( nullptr, 0, nullptr, nullptr );
```

Comment:
Did you mean 
```cpp
saved_pockets.emplace_back( nullptr, 0, nullptr );
```
here?

Reply:
yes I did thanks, I pulled this out of some bigger changes I decided to scrap 😆